.azure-piplines/master.yml

```diff
@@ -1,7 +1,13 @@
 trigger:
-  - master
+  branches:
+    exclude:
+      - "*"
 
-pr: none
+pr:
+  autoCancel: true
+  branches:
+    include:
+      - "*"
```

Comment:
Is this correct?

Reply:
I've changed this temporarily so that the master pipeline will trigger of this PR so that things can be tested. Should not be merged in this form though.